src/neo/SmartContract/ContainerPlaceholder.cs

```diff
@@ -14,7 +14,7 @@ public ContainerPlaceholder(StackItemType type, int count)
             ElementCount = count;
         }
 
-        public override bool Equals(object obj) => throw new NotSupportedException();
+        public override bool Equals(StackItem other) => throw new NotSupportedException();
```

Comment:
It's not better to compare with the same type like https://github.com/neo-project/neo/pull/1711 ?

Reply:
These methods are never used.